make-options.json

```diff
@@ -3,6 +3,7 @@
         "ANT",
         "Gulp",
         "NuGetInstaller",
+        "PublishSymbols",
```

Comment:
java-common/typings/globals/node/index.d.ts is missing
it's on my disk but when I do git status it doesn't show .... hmmm
I can add but trying to figure out why.  looking at gitignore etc...